src/Lumper.Lib/Jobs/RemoveAssetJob.cs

```diff
@@ -102,4 +107,22 @@ public override bool Run(BspFile bsp)
             return false;
         }
     }
+
+    private static void RemoveStaticProp(BspFile bsp, string path)
+    {
+        Sprp? sprp = bsp.GetLump<GameLump>().GetLump<Sprp>();
+        if (sprp is null)
+            return;
+
+        int idx =
```

Comment:
I keep forgetting how this works, but does the `StaticPropsDict` null conditional short circuit the call to `FindIndex`?